tests/ApplicationFeeTest.php

```diff
@@ -2,92 +2,70 @@
 
 namespace Stripe;
 
-class ApplicationFeeTest extends TestCase
+define('TEST_RESOURCE_ID', 'fee_123');
+define('TEST_FEEREFUND_ID', 'fr_123');
+
+class ApplicationFeeTest extends StripeMockTestCase
 {
-    public function testUrls()
```

Comment:
@ob-stripe Flagging that `ApplicationFee::update()` exists even though it's unsupported in the API. Likely want to remove from the code.

Reply:
👍